### PR TITLE
Add documentation for each service

### PR DESCRIPTION
**Pull request by gianarb**
I would like to highlight what each service does, briefly but a bit more
detailed compared with the single line we have in the docs homepage.

I created a page for each service (excluded pbnj for now), I hope it
will be also a way to start getting more of it written for each service.

Signed-off-by: Gianluca Arbezzano <gianarb92@gmail.com>

### CODE REVIEW

File: content/docs/services/hegel.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+```
+
+If you look at the `hardware-data.json` that we used during the Vagrant Setup
+you will find the `facility_code=onprem` as well. That's where the metadata
+comes from.  You can use the same format AWS EC2 uses, to get the meta-data
+index:
+
+```
+$ curl -s 192.168.1.2/2009-04-04/meta-data
```
Comment (DailyAlice) [inline content/docs/services/hegel.md:58]: is the distinction between `/metadata` and `meta-data` here important? Or is it a typo?

Comment (gianarb) [reply content/docs/services/hegel.md:58]: `/metadata` is the format hegel implement. `/meta-data` is the compatible format with AWS EC2. I placed it there as a reference "oh ec2 metadata works!"

Comment (DailyAlice) [reply content/docs/services/hegel.md:58]: Ah, so our metadata is also versioned with the date as well? As someone who knew very little about EC2 metadata (before googling it), I'm thinking I'll add some words explaining this format.

File: content/docs/services/tink.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+undefined TINKERBELL_CERT_UR
+```
+
+`undefined TINKERBELL_CERT_UR` notifies you about an authentication issue you
+are facing with the `tink-server`. All the traffic between tinkerbell services is encrypted via TLS.
+There are two environment variable that we have to use in order to authenticate
+the CLI:
+
+- `TINKERBELL_CERT_URL=http://127.0.0.1:42114/cert`
```
Comment (DailyAlice) [inline content/docs/services/tink.md:141]: is there a general, non-Vagrant form of this? It does look like the ports would be the same in a non-Vagrant setup, but what about the IP Address?

Comment (gianarb) [reply content/docs/services/tink.md:141]: This works everywhere, it explains how you can authenticate the CLI against a Tink-Server

Comment (DailyAlice) [reply content/docs/services/tink.md:81]: Cool, thanks. I edited this section to be a bit more general, let me know if it's still correct.

Comment (gianarb) [reply content/docs/services/tink.md:81]: It works!

**Comment by DailyAlice** [comment]
So it turns out I don't know how to get the pages to display. Any ideas?

**Comment by gianarb** [comment]
I don't know what you mean I think :) where do you want them?

**Comment by DailyAlice** [comment]
Ah, I see. I tend to be of the opinion that everything should appear in the table of contents so it's findable no matter what page of the documentation you are currently on. Thanks!

**Comment by gianarb** [comment]
Thanks a lot @DailyAlice for your help!

**Pull request merged**